benchmarks/benchmarks/ci/esmf_regridder.py

```diff
@@ -37,3 +41,45 @@ def time_make_grid(self):
         esmf_grid.data[:] = 0
 
     time_make_grid.version = 1
+
+
+class TimeRegridding:
+    params = ["similar", "large source", "large target", "mixed"]
+
+    def setup(self, type):
+        lon_bounds = (-180, 180)
+        lat_bounds = (-90, 90)
+        n_lons_src = 20
+        n_lats_src = 40
+        n_lons_tgt = 20
+        n_lats_tgt = 40
+        h = 100
+        if type == "large source":
+            n_lons_src = 100
+            n_lats_src = 200
+        if type == "large target":
+            n_lons_tgt = 100
+            n_lats_tgt = 200
+        if type == "mixed":
+            coord_system_src = RotatedGeogCS(0, 90, 90)
+        else:
+            coord_system_src = None
+        grid = _grid_cube(
+            n_lons_src,
+            n_lats_src,
+            lon_bounds,
+            lat_bounds,
+            coord_system=coord_system_src,
+        )
+        tgt = _grid_cube(n_lons_tgt, n_lats_tgt, lon_bounds, lat_bounds)
+        src_data = np.arange(n_lats_src * n_lons_src * h).reshape(
+            [n_lats_src, n_lons_src, h]
```

Comment:
I assume the switch in ordering from lon-lat to lat-lon is deliberate to make sure that some regridding work is required? Otherwise it's unnecessarily confusing.

Reply:
I think this might be a hang over from somewhere in CF refering to dimensions in the reverse order to how they are described in numpy. I can't remember why it is this way, but i believe this is inherited from how it is done in other tests. I might have another look into if this still makes sense.